arelle/XmlValidateParticles.py

```diff
@@ -9,9 +9,13 @@
                                    ModelAll, ModelChoice, ModelSequence, 
                                    ModelAny, anonymousTypeSuffix)
 from arelle.ModelObject import ModelObject, ModelAttribute
+from arelle.XbrlConst import xsd
 from arelle.XmlValidate import validate
 
+
 def validateElementSequence(modelXbrl, compositor, children, ixFacts, iNextChild=0):
+    if compositor.modelDocument.targetNamespace == xsd:
+        return (iNextChild, True, None, None)
```

Comment:
There's no function doc, so I'm not sure exactly what we should be returning, but I wonder if plain/non-tuple `None` would be more accurate since we aren't actually validating anything.  It looks like `XmlValidate` skips if the result is `None`.

Reply:
There is a recursive call [here](https://github.com/Arelle/Arelle/blob/master/arelle/XmlValidateParticles.py#L52) that prohibits returning anything other than a tuple.